Specifications/Tool/com.apple.commands.built-in.validate.xcspec

```diff
@@ -0,0 +1,26 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+{
+    Type = Tool;
+    Identifier = com.apple.build-tools.platform.validate;
+    Name = Validate;
+
+    CommandLine = "builtin-validationUtility [options] [input]";
+
+    Outputs = (
+        "$(TEMP_FILE_DIR)/com.apple.build-tools.platform.validate"
+    );
+
+    CommandOutputParser = (
+        ("()()error:(.*$)", emit-error),
+        ("()()warning:(.*$)", emit-warning),
+    );
+}
```

Comment:
No options spring to mind here, but an example run (https://github.com/fastlane/fastlane/issues/3648) shows an additional `-validate-for-store` flag passed in. Maybe there's an option for it, or should be added in code somewhere? Add a TODO if not sure, I guess.